crates/acir/src/fallback/mod.rs

```diff
@@ -0,0 +1,93 @@
+use noir_field::FieldElement;
+
+use crate::{
+    circuit::{gate::Directive, Gate},
+    native_types::{Expression, Witness},
+};
+
+// Perform bit decomposition on the provided expression
+pub fn split(
+    gate: Expression,
+    bit_size: u32,
+    mut num_witness: u32,
+    new_gates: &mut Vec<Gate>,
+) -> Vec<Witness> {
+    let mut w_result = Vec::new(); //bit vector
```

Comment:
Lets rename it to bit_vector and remove the comment

Reply:
Done